phoenix-read-write/src/main/java/com/cloudera/cod/examples/sql/AbstractSqlClient.java

```diff
@@ -95,4 +100,144 @@ public void runWrites(String tableName, Connection conn, int numRecords) throws
       conn.setAutoCommit(prevAutoCommit);
     }
   }
+
```

Comment:
The new methods are only from TransactionalClient.
Wouldn't it make those sense to put them there ?

Reply:
Done.